src/arrayMethodJoin.js

```diff
@@ -4,8 +4,25 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    let stringFromThis = '';
+    const pseudoSeparator = separator;
+
+    for (const n of this) {
+      let item = n;
+
+      if (n === undefined || n === null) {
+        item = '';
+      }
+
+      stringFromThis += item;
```

Comment:
Or you can just don't add nothing if this element is null or undefined
(adding empty string and doing nothing will have same result)